lang/en/tool_advancedreplace.php

```diff
@@ -92,6 +92,8 @@
 $string['replacepageheader'] = 'Replace text stored in the DB';
 $string['replacepagename'] = 'Replace strings in the Database';
 $string['replace'] = 'Replace';
+$string['replacecheck'] = 'Are you sure you want to replace strings in the Database using the uploaded csv file?';
```

Comment:
ideally this shows vaguely what it is about to do, eg 'we are about to replace 12,456 records - are you sure?'